internal/eventstore/models/event.go

```diff
@@ -0,0 +1,88 @@
+package models
+
+import (
+	"encoding/json"
+	"reflect"
+	"time"
+
+	"github.com/caos/zitadel/internal/errors"
+)
+
+type EventType string
+
+func (et EventType) String() string {
+	return string(et)
+}
+
+type Event struct {
+	ID               string
+	Sequence         uint64
+	CreationDate     time.Time
+	Type             EventType
+	PreviousSequence uint64
+	Data             []byte
+
+	AggregateID      string
+	AggregateType    AggregateType
+	AggregateVersion Version
+	EditorService    string
+	EditorUser       string
+	EditorOrg        string
```

Comment:
I think we dont need editororg

Reply:
I think it's nice if we have the date from beginning and it's easy to collect them